tests/unit/hive_metastore/test_tables.py

```diff
@@ -107,3 +112,29 @@ def test_tables_returning_error_when_describing():
     tc = TablesCrawler(backend, "default")
     results = tc._crawl()
     assert len(results) == 1
+
+
+def test_skip_happy_path(mocker, caplog):
+    ws = mocker.patch("databricks.sdk.WorkspaceClient.__init__")
+    sbe = mocker.patch("databricks.labs.ucx.framework.crawlers.StatementExecutionBackend.__init__")
```

Comment:
use `MockBackend` to verify the right queries are issued.

Reply:
Done